.eslintrc.json

```diff
@@ -1,10 +1,29 @@
 {
   "extends": ["airbnb-base", "prettier", "plugin:node/recommended"],
   "plugins": ["prettier"],
+  "parserOptions": {
+    "ecmaVersion": 2020,
+    "sourceType": "module"
+  },
+  "settings": {
+    "import/resolver": {
+      "node": {
+        "extensions": [".js"]
+      }
+    }
+  },
   "rules": {
     "prettier/prettier": "error",
     "no-console": "off",
     "no-process-exit": "off",
-    "node/no-unpublished-require": "off"
+    "node/no-unpublished-require": "off",
+    "node/no-unpublished-import": "off",
```

Comment:
Needed to disable this and ES syntax rules for now, they're not fully supporting ESM syntax yet. From [docs](https://github.com/mysticatea/eslint-plugin-node/blob/11d2d41ccabb68e5a6b7624db4ec3944dce7c442/docs/rules/no-unpublished-import.md):
>⚠️ ECMAScript 2015 (ES6) does not define the lookup logic and Node does not support modules yet. So this rule spec might be changed in future.